xml/libvirt_configuration_virsh.xml

```diff
@@ -0,0 +1,1231 @@
+    &lt;capability type='phys_function'&gt;
+      &lt;address domain='0x0000' bus='0x04' slot='0x00' function='0x0'/&gt;
+    &lt;/capability&gt;
+  &lt;/capability&gt;
+&lt;/device&gt;</screen>
+     <para>
+      The following data is needed for the next step:
+     </para>
+     <itemizedlist>
+      <listitem>
+       <para>
+        <literal>&lt;domain&gt;0&lt;/domain&gt;</literal>
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        <literal>&lt;bus&gt;4&lt;/bus&gt;</literal>
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        <literal>&lt;slot&gt;16&lt;/slot&gt;</literal>
+       </para>
+      </listitem>
+      <listitem>
+       <para>
+        <literal>&lt;function&gt;0&lt;/function&gt;</literal>
+       </para>
+      </listitem>
+     </itemizedlist>
+    </step>
+    <step>
+     <para>
+      Create a temporary XML file (for example
+      <filename>/tmp/vf-interface.xml</filename> containing the data
+      necessary to add a VF network device to an existing VM Guest. The
+      minimal content of the file needs to look like the following:
+     </para>
+<screen>&lt;interface type='hostdev'&gt;<co xml:id="sriov.iface"/>
+ &lt;source&gt;
+  &lt;address type='pci' domain='0' bus='11' slot='16' function='0'2/&gt;<co xml:id="sriov.data"/>
+ &lt;/source&gt;
+&lt;/interface&gt;</screen>
+     <calloutlist>
+      <callout arearefs="sriov.iface">
+       <para>
+        VFs do not get a fixed MAC address; it changes every time the host
+        reboots. When adding network devices the <quote>traditional</quote>
+        way with <tag>hostdev</tag>, it would require to reconfigure the
+        VM Guest's network device after each reboot of the host, because of
+        the MAC address change. To avoid this kind of problem, libvirt
+        introduced the <tag class="attvalue">hostdev</tag> value,
+        which sets up network-specific data <emphasis>before</emphasis>
+        assigning the device.
+       </para>
+      </callout>
+      <callout arearefs="sriov.data">
+       <para>
+        Specify the data you acquired in the previous step here.
+       </para>
+      </callout>
+     </calloutlist>
+    </step>
+    <step>
+     <para>
+      In case a device is already attached to the host, it cannot be
+      attached to a guest. To make it available for guests, detach it from
+      the host first:
+     </para>
+<screen><prompt>tux &gt; </prompt><command>sudo virsh nodedev-detach pci_0000_04_10_0</command></screen>
+    </step>
+    <step>
+     <para>
+      Add the VF interface to an existing VM Guest:
+     </para>
+<screen><prompt>tux &gt; </prompt><command>sudo virsh attach-device <replaceable>GUEST</replaceable> /tmp/vf-interface.xml --<replaceable>OPTION</replaceable></command></screen>
+     <para>
+      <replaceable>GUEST</replaceable> needs to be replaced by the domain
+      name, id or uuid of the VM Guest and
```

Comment:
'id or uuid' -> 'ID' or 'UUID' (?).
Split into two sentences: 
'[...] of the VM Guest.  '--<replaceable>OPTION</replaceable> can be one of the following: [...]'